pkg/oc/cli/util/clientcmd/factory.go

```diff
@@ -91,7 +91,7 @@ func (f *Factory) PrintResourceInfos(cmd *cobra.Command, isLocal bool, infos []*
 	}
 
 	printAsList := len(infos) != 1
-	object, err := AsVersionedObject(infos, printAsList, schema.GroupVersion{}, legacyscheme.Codecs.LegacyCodec())
+	object, err := AsVersionedObject(infos, printAsList, schema.GroupVersion{Version: "v1"}, legacyscheme.Codecs.LegacyCodec())
```

Comment:
This smells wrong. Will reconsider tomorrow, but can you post before and after errors and output?